### ISSUE TITLE
Option to suppress the message "Starting Opium on ...." 

### ISSUE THREAD

**Issue by Nymphium** [rgrinberg/opium#260]

**Comment by tmattio** [comment]
Hi, could you describe the problem you have with the message "Starting Opium on"?

**Comment by aantron** [comment]
I have my own problem with this message, which is that I'd like to replace it with a URL that will be clickable in the terminal, which I emit myself because it is based on app configuration. So I'd like to suppress this message as it is always either misleading or redundant in such an app.

Also, it, and, apparently, all the other log expressions in `app.ml`, print to the application logger. This is wrong because there is no good way to selectively suppress them and still retain easy logging in the true, user's application. The Opium `App` is, of course, still a library, not an application.

This is also the reason why Logs itself [recommends](https://erratique.ch/software/logs/doc/Logs/index.html#usage) against libraries logging with the default source. Opium has an `opium.server` source somewhere that could be used for some of these prints, but the "starting" print should be removed or at least suppressed anyway, because there are legitimate reasons why a user would want to see the other messages but not "starting."

**Comment by anuragsoni** [comment]
> Also, it, and, apparently, all the other log expressions in `app.ml`, print to the application logger. This is wrong because there is no good way to selectively suppress them and still retain easy logging in the true, user's application. The Opium `App` is, of course, still a library, not an application.

Agreed. Opium shouldn't log to the default source. I'll take a look at all the places opium logs content and make sure that we only use an opium specific log source.

**Comment by Nymphium** [comment]
Others already said the reason why needed the option. It's better to control where to display or what to say any messages in server by library users.
